grails-app/migrations/core/2021-05-14-stream-fk-storage-node.groovy

```diff
@@ -1,15 +1,42 @@
 package core
 
+import com.unifina.utils.IdGenerator
+
 databaseChangeLog = {
 	changeSet(author: "kkn", id: "stream-fk-storage-node-1") {
-		addForeignKeyConstraint(
-			constraintName: "fk_stream_has_storage_nodes",
+		sql("alter table stream_storage_node drop primary key")
+		sql("alter table stream_storage_node drop index stream_storage_node_stream_idx")
```

Comment:
Do we need to drop these indexes and then create same indexes at lines 29, 30?

Reply:
Might not be strictly necessary, but it makes the DB rebuild the index. (Granted that in a DB of our size it probably isn't required.)